### DISCUSSION TITLE
borgbackup - msgpack error

### DISCUSSION THREAD

**Discussion by dajiahao** [Entware/Entware#1032] [Q&A]
I have borgbackup and msgpack installed

I think the related packages are [borgbackup_1.2.4-4_armv7-3.2.ipk] and [python3-msgpack_1.0.7-1_armv7-3.2.ipk]

> borg -V    
borg 1.2.4

but backup to this machine doesn't work, I always get an error about msgpack

> borg serve --info
$LOG ERROR borg.archiver Remote: You do not have a supported version of the msgpack python package i
nstalled. Terminating.
$LOG ERROR borg.archiver Remote: This should never happen as specific, supported versions are requir
ed by our setup.py.
$LOG ERROR borg.archiver Remote: Do not contact borgbackup support about this.

any idea how to fix this? I checked for doubled msgpacks and also reinstalled all python related parts

**Comment by ryzhovau** [comment]
Can't reproduce it on 1.4.0, sorry.
```
~ # borg serve --info

$LOG ERROR borg.archiver Remote: Borg 1.4.0: Got unexpected RPC data format from client.
~ # borg --version

borg 1.4.0
```
